api.jsx

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: "https://my-nc-news-s3ph.onrender.com/api",
+});
+
+export function getArticles() {
+  return api.get(`/articles`).then(({ data }) => {
+    return data;
```

Comment:
You could return the articles array from the api function rather than the whole object